tools/installers/install_pyopenjtalk.sh

```diff
@@ -13,7 +13,8 @@ if [ ! -e pyopenjtalk.done ]; then
         set -euo pipefail
         # Since this installer overwrite existing pyopenjtalk, remove done file.
         [ -e tdmelodic_pyopenjtalk.done ] && rm tdmelodic_pyopenjtalk.done
-        python3 -m pip install pyopenjtalk==0.3.0
+        # NOTE(kan-bayashi): Should update after new version release
+        python3 -m pip install --no-build-isolation pyopenjtalk==0.3.0
```

Comment:
Hi, thanks for the update :) I've released a new version 0.3.1: https://github.com/r9y9/pyopenjtalk/releases/tag/v0.3.1

Reply:
Thank you GOD